src/Ucommerce.Transactions.Payments.QuickpayLink/QuickpayPaymentMethodService.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using Ucommerce.EntitiesV2;
+using System.Web;
+using Ucommerce.Extensions;
+using Ucommerce.Web;
+using Ucommerce.Transactions.Payments.Common;
+
+namespace Ucommerce.Transactions.Payments.QuickpayLink
+{
+    public class QuickpayPaymentMethodService : ExternalPaymentMethodService
+    {
+        private string ApiKey(PaymentMethod paymentMethod) => paymentMethod.DynamicProperty<string>().ApiKey;
+        private string CallbackUrl(PaymentMethod paymentMethod) => paymentMethod.DynamicProperty<string>().CallbackUrl;
+        private string AcceptUrl(PaymentMethod paymentMethod) => paymentMethod.DynamicProperty<string>().AcceptUrl;
```

Comment:
This does not look like it is being used since you are doing the same fetch of the property on line 71

Reply:
Removed unused variable.